src/components/ViewerTitle/index.tsx

```diff
@@ -44,7 +44,7 @@ const ViewerTitle: React.FunctionComponent<ViewerTitleProps> = (
         lastModified && lastModified - Date.now() < 0;
     if (shouldShowModifiedDate) {
         tagText = `modified: ${moment(lastModified).format(
-            "YYYY-MM-DD, h:m A"
+            "YYYY-MM-DD, h:mm A"
```

Comment:
Unrelated change that shows "6:09 PM" instead of "6:9 PM" for example for last modified time